tests.js

```diff
@@ -10,7 +10,7 @@ let textUnicode = require("ot-text-unicode");
 var jsondiff = require("./index.js");
 
 var tests = [
-  //tests of equality
+	// Tests of equality
```

Comment:
Thanks! It might be nice to [adopt Prettier](https://github.com/kbadk/json0-ot-diff/issues/23) so these things don't come up in the future.